cli/src/commonMain/kotlin/StartCommand.kt

```diff
@@ -62,9 +63,27 @@ import org.eclipse.apoapsis.ortserver.cli.utils.useJsonFormat
 import org.eclipse.apoapsis.ortserver.client.NotFoundException
 import org.eclipse.apoapsis.ortserver.client.OrtServerClient
 
-internal val POLL_INTERVAL = getEnv("POLL_INTERVAL")?.toLongOrNull()?.seconds ?: 10.seconds
+/**
+ * The maximum number of failures in a sequence that cause the waiting loop for a run to finish to abort.
+ */
+private const val MAX_FAILURE_COUNT = 10
 
 class StartCommand : SuspendingCliktCommand(name = "start") {
+    companion object {
+        /**
+         * The interval to poll the server for updates when waiting for a run to finish. This is modifiable to allow
+         * tests to run faster.
+         */
+        private var pollInterval = 10.seconds
```

Comment:
How about making this a command line option instead, and using that in the test?

Reply:
We deliberately decided against this in the past, since we do not want to allow users to set the polling interval to a few nanoseconds.